web/templates/compose.hbs

```diff
@@ -76,14 +74,20 @@
                     </div>
                     <div class="messagebox-wrapper">
                         <div class="messagebox">
-                            <textarea class="new_message_textarea" name="content" id='compose-textarea' placeholder="{{t 'Compose your message here' }}" tabindex="0" aria-label="{{t 'Compose your message here...' }}"></textarea>
-                            <div id="preview-message-area-container">
-                                <div class="scrolling_list preview_message_area" data-simplebar id="preview_message_area" style="display:none;">
-                                    <div class="markdown_preview_spinner"></div>
-                                    <div class="preview_content rendered_markdown"></div>
+                            <div id="message-content-container">
+                                <textarea class="new_message_textarea" name="content" id='compose-textarea' placeholder="{{t 'Compose your message here' }}" tabindex="0" aria-label="{{t 'Compose your message here...' }}"></textarea>
+                                <div id="preview-message-area-container">
+                                    <div class="scrolling_list preview_message_area" data-simplebar id="preview_message_area" style="display:none;">
+                                        <div class="markdown_preview_spinner"></div>
+                                        <div class="preview_content rendered_markdown"></div>
+                                    </div>
                                 </div>
+                                <div class="composebox-buttons">
```

Comment:
So question: will you be able to reuse this `.compose-box` DIV for the formatting buttons? If not, it might be better to choose a more specific class name here--though I guess `composebox-formatting-buttons` is the likely candidate for those. Just something to think about, especially given the grid that you're setting up on this.

Reply:
Yeah, I am not really able to come up with a better name for these... My thinking here was that these are compose *box* buttons while the formatting ones are compose *control* buttons.

Do you have any better suggestions in mind?

Reply:
Maybe `composebox-textarea-side-buttons` would be good.